modules/auxiliary/scanner/http/owa_login.rb

```diff
@@ -79,7 +79,7 @@ def initialize
     register_options(
       [
         OptInt.new('RPORT', [ true, "The target port", 443]),
-        OptAddress.new('RHOST', [ true, "The target address", true]),
+        OptAddress.new('RHOST', [ true, "The target address", '']),
```

Comment:
Also, the empty string can technically be removed, since the option is required.